### ISSUE TITLE
A11y_VisualStudioCodeClient_Version Control_Screenreader:NVDA/Narrator is not announcing specific name for "Checkout branch/tag button"

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/vscode#159454]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG1.3.1; #NVDA; #ScreenReader; #Narrator; 

### Environment and OS details:
Visual Studio Code 
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:

1. Create branch by using this https://code.visualstudio.com/docs/editor/versioncontrol
2. Turn on NVDA
3. Open Visual studio code editor.
4. Open-source control using Ctrl+Shift+G
5. Verify that screen reader is announcing specific name for Checkout branch/tag..., button, or not.

### Actual Result:
NVDA/Narrator is not announcing specific name for "Checkout branch/tag button".

### Expected Result:
NVDA/Narrator should announce specific name for "Checkout branch/tag button".

### Attachment:

![Narrator](https://user-images.githubusercontent.com/100189407/187200705-2cfaa762-bb3f-49af-a4dd-90b142e7945b.jpg)
![Nvda ](https://user-images.githubusercontent.com/100189407/187200713-f6c6733c-7b3c-44de-ba3e-cd2b6b196cc9.jpg)

**Comment by isidorn** [comment]
@VijayaBharathyA thanks for filling this.
@lszomoru I think this issue makes sense. So the title of this button should also contain the branch name
Code pointer https://github.com/microsoft/vscode/blob/main/extensions/git/src/statusbar.ts#L32

Similar to how you compute the title. Let me know what you think.

**Comment by lszomoru** [comment]
@isidorn, sounds good to me.
